notebooks/binary_predicates.ipynb

```diff
@@ -0,0 +1,1157 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.            'GeoPandas predicate time',
</pre><p> </p><p>I have forgotten the difference between "predicate time" and "time". You said at one point you want to remove one of them?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cuspatial/pull/1156/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>